### DISCUSSION TITLE
Negative splitratio not working?

### DISCUSSION THREAD

**Discussion by guttermonk** [hyprwm/Hyprland#7444] [General]
Did the negative splitratio stop working for anyone else?
`bind = Super Shift, M, exec, hyprctl dispatch splitratio -0.1`

What's odd is that the positive splitratio works fine.
`bind = Super, M, exec, hyprctl dispatch splitratio 0.1`

Can anyone confirm?

**Comment by vaxerski** [comment]
`hyprctl -- dispatch splitratio -0.1`

**Comment by guttermonk** [comment]
Thank you!
